mmv1/third_party/terraform/services/bigtable/resource_bigtable_table.go

```diff
@@ -152,6 +175,29 @@ func resourceBigtableTableCreate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 
+	if automatedBackupPolicyField, ok := d.GetOk("automated_backup_policy"); ok {
+		automatedBackupPolicy := automatedBackupPolicyField.(map[string]interface{})
+		abpRetentionPeriodField, retentionPeriodExists := automatedBackupPolicy["retention_period"]
+		if !retentionPeriodExists {
+			return fmt.Errorf("Automated backup policy retention period must be specified")
+		}
+		abpFrequencyField, frequencyExists := automatedBackupPolicy["frequency"]
+		if !frequencyExists {
+			return fmt.Errorf("Automated backup policy frequency must be specified")
+		}
+		abpRetentionPeriod, err := time.ParseDuration(abpRetentionPeriodField.(string))
+		if err != nil {
+			return fmt.Errorf("Error parsing automated backup policy retention period: %s", err)
+		}
+		abpFrequency, err := time.ParseDuration(abpFrequencyField.(string))
+		if err != nil {
+			return fmt.Errorf("Error parsing automated backup policy frequency: %s", err)
+		}
+		tblConf.AutomatedBackupConfig = &bigtable.TableAutomatedBackupPolicy{
+			RetentionPeriod: abpRetentionPeriod,
+			Frequency:       abpFrequency}
```

Comment:
this looks like weird formatting. I'd have abpFrequency, and } on a new line

Reply:
Thanks, updated